aten/src/ATen/native/VariableMethodStubs.cpp

```diff
@@ -0,0 +1,16 @@
+#include <ATen/ATen.h>
+#include <ATen/NativeFunctions.h>
+
+namespace at {
+namespace native {
+
+void backward(const Tensor& self, const Tensor& gradient, bool keep_graph, bool create_graph) {
+  AT_ERROR("backward is not implemented for Tensor");
+}
+
+void set_data(const Tensor& self, const Tensor& new_data) {
```

Comment:
the codegen for this looks wrong, because this is really an inplace method, so you'd think `self` would be `Tensor &`, not `const Tensor &`, probably because this doesn't end in a _.  I can't off the top of my head think why this would cause a problem, just something to note.